### DISCUSSION TITLE
Exclude paths

### DISCUSSION THREAD

**Discussion by eldar** [Glyphack/enderpy#258] [Q&A] [answered]
Hi! I am interested in trying out Enderpy as a Python LSP. Currently I use Pyright which I can configure to exclude certain paths from being scanned in `pyproject.toml` - such as the `.venv` which I believe should be scanned automatically if its interpreter is the first on the `PATH`. Does Enderpy have a similar setting? Thanks!

**Comment by Glyphack** [comment, marked as answer]
Hey, Enderpy is not ready for usage since We're still working on the core functionality of the type checker. LSP clients are for testing and debugging right now. The setting you described is needed definitely but right now I don't have the bandwidth to work on it. But I'll add it once we are there.

**Reply by eldar** [reply]
Thanks for clarifying, I'll check on Enderpy later when more of the core functionality is ready! Good luck with this project, it is looking very promising!
